evolver/tests/conftest.py

```diff
@@ -1,6 +1,13 @@
 import pytest
 
+from evolver.settings import settings
+
 
 @pytest.fixture
 def testme():
     return "value"
+
+
+@pytest.fixture
+def tmp_calibration_dir(monkeypatch, tmp_path):
```

Comment:
might be useful as an autouse?

Reply:
Actually, this doesn't work as the fixture order matters.